src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -198,13 +216,14 @@ applyGenesisCmd
       -- ^ Contains block height, time, prev hash + metadata
     -> SPVSupport
       -- ^ SPV support (validates cont proofs)
-    -> Command (Payload PublicMeta ParsedCode)
+    -> Command PayloadWithText
       -- ^ command with payload to execute
     -> IO (T2 (CommandResult [TxLog Value]) ModuleCache)
-applyGenesisCmd logger dbEnv pd spv cmd = do
+applyGenesisCmd logger dbEnv pd spv cmdIn = do
```

Comment:
this should be unchanged as we don't charge gas on genesis

Reply:
see latest commit